Request (Insomnia) Unmanaged and manage user explanation (#3744)
Kong/developer.konghq.com: app/insomnia/enterprise-user-management.md

This data is maintained for 90 days. After 90 days without any activity, the date no longer appears in the report but the user will always be listed. If a user was last active more than 100 days before the report was generated, the **Last Active** value for that user will be `N/A` in the report. This allows you to quickly find inactive users.

{:.info}
> For any user who last logged in before May 20, 2025, the date will show their last login date.

## User types

In your Enterprise Plan, there are two types of users:
- **Managed Users**: A user that operates Insomnia from one of your verified domains and is associated with your Enterprise account. These users consume a license.
- **Unmanaged Users**: A user that operates Insomnia from one of your verified domains, but is not associated with your Enterprise account.

View all of the Unmanaged Users associated with a domain:
1. From Insomnia [**Enterprise Controls**](https://app.insomnia.rest/app/enterprise/), click **Domains**.
1. Select your domain.
1. From the **Unclaimed Accounts** section, click **Manage**.

{:.decorative}

> > Tip: Keep all users associated with your Enterprise account to avoid compliance breaches and security risks.

Reviewer: Is there an option to not keep then associated with your enterprise account?

> > **Tip:** Keep all users associated with your Enterprise account to avoid compliance breaches and security risks.

Author: Okay, this is excellent insight. I think a good way to go is to select option 1, for this git issue. So i'll remove the tip, and keep the information informative and unbiased.

However, I see the insight in options 2/3 better applied to a Domain Capture / Domain Lock specific Issue - So I will create a new issue to uplift Domain Capture, and Domain Lock with these insights, and user journey focus.

To associate Unmanaged Users with your Enterprise account, enable [Domain Capture]():
1. From Insomnia [**Enterprise Controls**](https://app.insomnia.rest/app/enterprise/), click **Domains**.
1. Select your domain.
1. Activate the **Enable** toggle.

{:.info}

> > If you have more than one domain, the number of Unmanaged Users will not match the number of Unclaimed Accounts. Unmanaged Users offers a cumulative view of all of the Unmanaged Users spread across all of your domains. When you view the Unclaimed Accounts that is associated with a domain, you are viewing only the Unmanaged Users associated with that specific domain.

Reviewer 2: ```suggestion
> The number of unmanaged users is cumulative across all your domains. Unclaimed accounts are specific to one domain and only reflect the unmanaged users associated with that specific domain. As a result, if you have more than one domain, the number of unmanaged users won't match the number of unclaimed accounts.
```
I tried to revise this to highlight how both are calculated to explain why the numbers are different. I think this is what the original was trying to say, but let me know if that's the incorrect assumption.
